src/Cookie/CookieJar.php

```diff
@@ -22,7 +22,7 @@ class CookieJar implements CookieJarInterface
      *                           arrays that can be used with the SetCookie
      *                           constructor
      */
-    public function __construct($strictMode = false, $cookieArray = [])
+    public function __construct($strictMode = false, array $cookieArray = [])
```

Comment:
This is technically a BC break.. Or i mean. It has always been the intention but people will have problem if we merge this.

Reply:
What I had in mind:

1. Constructor is allowed to be overridden in PHP and it is not required to follow parent.
2. If some has passed a different value rather than array it would still have failed on foreach...

but now that I think of it.. in cases where someone was passing an `iterable` this will fail...  So yea I will revert. 

😢 👍

Reply:
Reverted... resolving.